pkg/tests/cli_test.go

```diff
@@ -0,0 +1,227 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"bytes"
+	"crypto"
+	"errors"
+	"io"
+	"math/big"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	ts "github.com/digitorus/timestamp"
+	"github.com/sigstore/timestamp-authority/pkg/client"
+	"github.com/sigstore/timestamp-authority/pkg/generated/client/timestamp"
+)
+
+const (
+	cli = "../timestamp-cli"
```

Comment:
`..` or `.`?

Reply:
It's been updated to `../..` since the binary is built in the root dir. But we can create another target that is built in the pkg/tests dir

Reply:
I had to add building the CLI in the testing GHA workflow to get everything passing. Were you thinking of a different approach to this?